_data/teams.yml

```diff
@@ -44,3 +44,10 @@ Web Development:
 
 Security Engineering:
   lever: 'Security Engineering'
+
+Internal Tools:
+  lever: 'Internal Tools'
+  about: |
```

Comment:
Its not clear to me - where is this used? (I see that most other teams do not have an `about` defined here)

Reply:
Yea looks like the `about` is unused. I can remove it. The lever part I'm assuming is for the "Related Jobs" part at the bottom of each post https://tech.scribd.com/blog/2020/Recycle-EKS-Worker-Nodes.html.

Reply:
There was a hope to have more "about" the teams which never materialized in our early designs